.github/workflows/tests.yml

```diff
@@ -4,6 +4,8 @@ on:
   push:
     paths-ignore:
       - "*.md"
+    branches:
+      - "main"
```

Comment:
I'm not sure I'd add this? If you work in a fork instead (as we usually do), then it's not an issue?

Reply:
The tests are restricted to `explosion`, so they won't run by default in forks. This could be extended to the validate step, too?

Reply:
> This could be extended to the validate step, too?

I just pushed a commit, thanks.

Just to be sure, triggering the CI on every push is part of the general MO, right?

Reply:
Typically the pushes are from the PR branch in a fork to a branch in the explosion repo, either `main` or a release branch like `v0.1.x`. It's not that we couldn't specify the branches, but for the amount of traffic in this repo it sounds like more trouble to maintain than it's worth?

(I'm very annoyed about how GHA is often automatically enabled in forks. It didn't matter for me in particular, but I went through most of my "personal" minutes when some windows jobs hung for a PR branch for 6 hours in a fork where I never intended for it to be enabled.)

Reply:
Ah right. Ok, the check for repo ownership is in place.

And I've removed the branch constraint on push events. I think we could do something like:

```yaml
    branches:
      - "main"
      - "release/*"
```

to save a lot of resources and still get CI triggers on main and release candidates. Right now every push to a PR triggers two independent CI runs...

Reply:
Ok I finally got your point about forks. I guess I'm still not used to that MO.